ceph-dev-new-setup/build/build

```diff
@@ -86,6 +86,17 @@ else
   printf 'No cmake debug options added to branch %s.\n' "$BRANCH"
 fi
 
+# Tentacle is the last release that needs dedicated Crimson builds,
+# Later releases are able to use Crimson with the default build.
+# As the "Crimson flavor" is no longer available, we need a *temporary* way
+# to be able build Crimson for tentacle.
+# Note: This could be removed once Crimson we have Umbrella release builds.
```

Comment:
We will still need to be able to build Tentacle until upgrade testing from T to V is no longer necessary (probably when V is EOL).

Reply:
This section is only true for Crimson. We only build the last stable release (and main) as there's no upgrade testing/support yet. Starting U we will probably start working on upgradability. We still break on-disk data structures from time to time as we are still flexible with this - the plan is to stop with this approach starting U.
This is why we should be able to remove this bit once U is out as Crimson will no longer be supported in T _at all_. I'll mention this in this comment to avoid confusion.